### DISCUSSION TITLE
Since several times already me and my friends have mistakenly ended the round for all. I suggest rename "return to lobby" to "break round".

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14977] [Ideas and suggestions]
Since me and my friends who have administrator rights have already several times mistakenly ruined the game by ending the round for everyone while they just wanted to leave the server.
I think this is not only our mistake.

I suggest rename in main menu "return to lobby" to "break round".
Also in Russian "Выйти в лобби" rename to "Прервать раунд".  "Испортить раунд" may be even more appropriate

**Comment by SomeRandomNoobKekeke** [comment]
"Заруинить раунд"

**Comment by girlyguppy** [comment]
"break round" is way more confusing as a primarily english speaking person. I've never heard that phrase used.

The text "end round" is already used in mission and pvp modes, so I'm not sure why it isn't used in campaign. Campaign should definitely replace "return to lobby" with "end round", because I agree that "return to lobby" sounds like going back to the lobby without leaving the server.

Of course, the host cannot leave without ending the round

**Reply by pavel4win** [reply]
My native language is not English but I hope you understand what I'm talking about.
Peoples not so clever as we want they to be. Be better not just "end round". better is "end round and blow off all your progress".

**Reply by pavel4win** [reply]
I've misunderstood the phrase "Выйти в лобби" at least three times. And I don't consider myself a stupid person.
However, maybe this isn't something you should spend your workday on.

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
